### PR TITLE
Add CHANGELOG.md

### LINKED ISSUE #98: Add CHANGELOG.md
In anticipation of the [versioning guidelines](https://github.com/camaraproject/WorkingGroups/pull/123) from Commonalities there should be a CHANGELOG.md within the project.

Will generate an initial one from the release descriptions of v0.1.0 and v0.8.0.

Going forward a new release could be discussed and agreed by a pull request to the CHANGELOG.md. The merge commit of this pull request will be then tagged as the release (not for v0.8.0 as we are already beyond that point).

### PR DESCRIPTION
**Pull request by hdamker**
Will close #98

### CODE REVIEW

**Comment by hdamker** [comment]
Content for v0.1.0 and v0.8.0 copied from (partly autogenerated) release descriptions in Github.
Both release branches unfortunately haven't a CHANGELOG.md yet, I propose to forward correct that soon with a v0.8.1 release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+**This is the first alpha version of the Quality-On-Demand (QoD) API.** 
+- API [definition](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0/code/API_definitions)
+- API [documentation](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0/documentation/API_documentation)
+
+## Please note:
+
+- This is an alpha version, it should be considered as a draft.
+- There are bug fixes to be expected and incompatible changes in upcoming versions. 
+- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+- Version numbers 0.2.x to 0.7.x where used in privat versions during the development of the API and are here not used to avoid conflicts with local implementations.
```
Comment (SfnUser) [inline CHANGELOG.md:24]: s/where used in privat /were used in private /g

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/v0.1.0...v0.8.0
+
+# v0.1.0 - Initial contribution
+
+**Initial contribution of two API definitions for Quality on Demand (stable bandwidth and stable latency)**, including initial documentation and implementation code.
+
+## Please note 
+- this "release" is only tagged to document the history of the API, it is not intended to be used by implementors or API customers
+- it was implemented by Deutsche Telekom within lab environment and tested against two NEF implementations
+- going forward the implementation [code](https://github.com/camaraproject/QualityOnDemand/tree/50e81e0c4a6a7431c0a7f50c26415caf935be6df/code/API_code) will not be part of releases of QoD API. Instead it will be provided within seperate repositories (QualityOnDemand_PIx).
```
Comment (SfnUser) [inline CHANGELOG.md:49]: s/seperate/separate/g

File: CHANGELOG.md (hunk 1 above)
Comment (hdamker) [reply CHANGELOG.md]: Done in [19eba91](https://github.com/camaraproject/QualityOnDemand/pull/99/commits/19eba91458b220e458c2604b4a0a5fa03a50519d)

File: CHANGELOG.md (hunk 2 above)
Comment (hdamker) [reply CHANGELOG.md]: Done in [19eba91](https://github.com/camaraproject/QualityOnDemand/pull/99/commits/19eba91458b220e458c2604b4a0a5fa03a50519d)

Comment (SfnUser) [reply CHANGELOG.md]: sorry for being pedantic but could you please correct the typo 'privat' in line 24?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Changelog QualityOnDemand
+
+## Content
```
Comment (shilpa-padgaonkar) [inline CHANGELOG.md:3]:
```suggestion
## Table of Contents
```

**Review by jlurien — COMMENTED** [review]
I see that this more or less what we have also in <https://github.com/camaraproject/QualityOnDemand/releases>. For the current status of the project I think it is enough, but once we are closer to v1.0.0 we should have a detail section, or a separate file, with the changes in the API spec,. following the format that is being proposed in commonalities.

File: CHANGELOG.md (hunk 3 above)
Comment (hdamker) [reply CHANGELOG.md]: Done in [8ec7dba](https://github.com/camaraproject/QualityOnDemand/pull/99/commits/8ec7dba74e7a1a916a49c3439bfe4f328406fb7e)

**Review by shilpa-padgaonkar — APPROVED** [review]
/approve

**Comment by hdamker** [comment]
@jlurien yes, that is more or less the outcome of the Github release feature, content automatically generated there (What's changed / new contributor) + notes in beginning - chore or orga PRs. 

I agree to sort the changes going forward in categories, but for that we need also improve the information within the PRs, e.g. with a (simple) template. That might also allow to use tooling to reduce the work in release process.

**Pull request merged**